modules/core/src/main/java/org/apache/ignite/internal/processors/cache/verify/ViewCacheClosure.java

```diff
@@ -101,6 +105,17 @@ public ViewCacheClosure(String regex, VisorViewCacheCmd cmd) {
                     ci.setAtomicityMode(context.config().getAtomicityMode());
                     ci.setMapped(mapped(context.caches().iterator().next().name()));
 
+                    GridCacheProcessor cacheProcessor = k.context().cache();
+
+                    long sizeSummary = 0L;
+
+                    for (GridCacheContext cacheContext : context.caches()) {
+                        IgniteCache<Object, Object> cache = cacheProcessor.jcache(cacheContext.cache().name());
```

Comment:
check once more AI codestyle, section about new lines and logically code division.

Reply:
Fixed